bgpd/bgp_filter.h

```diff
@@ -34,6 +41,8 @@ struct as_list {
 
 	struct as_filter *head;
 	struct as_filter *tail;
+	/* back pointer to the aspath_excludes */
+	struct bp_aspath_exclude_list *bp_list;
```

Comment:
As I mentioned in my previous review, `bp_` seems absolutely useless. And `bp_list` is also "non-understandable" naming.